homeassistant/components/webhook/__init__.py

```diff
@@ -28,7 +31,13 @@
 
 @callback
 @bind_hass
-def async_register(hass, domain, name, webhook_id, handler):
+def async_register(
+    hass: HomeAssistant,
+    domain: str,
+    name: str,
+    webhook_id: str,
+    handler: Callable,
```

Comment:
We should make this more specific by defining a protocol, so we can check the signatures of the handler.